src/routes/oauth/config.ts

```diff
@@ -385,4 +385,25 @@ export const PROVIDERS_CONFIG: Record<
       next();
     },
   },
+  oidc: {
+    grant: {
+      oauth: 2,
+      nonce: true,
+      scope_delimiter: ' ',
+      scope: ['openid', 'profile', 'email'],
```

Comment:
Do all OIDC providers have these three scopes? Are there some official OIDC documentation around this?

Reply:
Yes:
- https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims
- https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes
- https://learn.microsoft.com/EN-US/azure/active-directory/develop/scopes-oidc